src/v0/destinations/criteo_audience/docs/businesslogic.md

```diff
@@ -0,0 +1,213 @@
+# Criteo Audience Business Logic
+
+## Overview
+
+This document outlines the business logic and implementation details of the Criteo Audience destination integration for managing audience segments.
+
+## Event Processing
+
+### Event Types
+
+The Criteo Audience destination only supports the `audiencelist` event type. This event type is used to manage audience segment members in Criteo.
+
+### Event Structure
+
+```javascript
+{
+  type: 'audiencelist',
+  properties: {
+    listData: {
+      add: [
+        { email: 'user1@example.com' },
+        { email: 'user2@example.com' }
+      ],
+      remove: [
+        { email: 'user3@example.com' }
+      ]
+    }
+  }
+}
+```
+
+### Event Validation
+
+1. **Required Fields**:
+
+   - `type`: Must be 'audiencelist'
+   - `properties`: Must be present
+   - `properties.listData`: Must be present
+   - `properties.listData.add` or `properties.listData.remove`: At least one must be present
+
+2. **Identifier Validation**:
+   - Each object in `add` or `remove` arrays must contain the identifier type specified in the configuration
+   - For GUM identifier type, `gumCallerId` must be provided in the configuration
+
+## Data Processing
+
+### Batch Processing
+
+1. **Chunking**:
+
+   - Identifiers are automatically chunked into batches of 50,000
+   - Each batch is processed as a separate request to the audience segment
+   - Chunking is handled by the `populateIdentifiers` function
+
+2. **Payload Structure**:
+   ```javascript
+   {
+     data: {
+       type: 'ContactlistAmendment',
+       attributes: {
+         operation: 'add' | 'remove',
+         identifierType: 'email' | 'maid' | 'gum' | 'identityLink',
+         internalIdentifiers: false,
+         gumCallerId: 'your-gum-caller-id', // Only for GUM type
+         identifiers: ['identifier1', 'identifier2', ...]
+       }
+     }
+   }
+   ```
+
+### Identifier Types
```

Comment:
Let's add some more information on the data types Criteo can support, or you can provide the documentation link here as well, if any are available.